Detections/MultipleDataSources/AdditionalFilesUploadedByActor.yaml

```diff
@@ -0,0 +1,95 @@
+id: b9e3b9f8-a406-4151-9891-e5ff1ddd8c1d
+name: Linked Malicious Storage Artifacts
+description: |
+  'An IP address which uploaded malicious content to an Azure Blob or File Storage container (triggering a malware alert) also uploaded additional files.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: MicrosoftCloudAppSecurity
+    dataTypes:
+      - SecurityAlert
+queryFrequency: 15m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+  - Exfiltration
+  - ResourceDevelopment
+relevantTechniques:
+  - T1584
+  - T1071
+  - T1567
+query: |
+
+  //Collect the alert events
+  let alertData = SecurityAlert 
+  | where TimeGenerated > ago(30m) 
```

Comment:
Queries are moving away from hard coding the lookback unless it is required based on different lookback timeframes.  In this case, we use the same 30mins.  This should be removed and the query will be controlled by the queryPeriod value in the YAML.